src/components/postDetail/PostDetailContent/PostDetailContent.module.scss

```
@@ -0,0 +1,66 @@
+.main-banner {
+  width: 100%;
+}
+
+.main-section {
+  @include responsive(T) {
+    padding: 0 4rem;
+  }
+
+  @include responsive(M) {
+    padding: 0 1.5rem;
+  }
+
+  width: 100%;
+
+  .container {
+    @include responsive(T) {
+      max-width: 100%;
+      padding-top: 3.2rem;
+    }
+
+    max-width: 112rem;
+    margin: 0 auto;
+    padding: 2.4rem 0 12rem;
+
+    .section-top {
+      @include column-flexbox(center, start, 1.6rem);
+
+      @include responsive(T) {
+        padding-bottom: 1.6rem;
+      }
+
+      width: 100%;
+      padding-bottom: 3.2rem;
+    }
+
+    .section-bottom {
+      @include flexbox(between, start);
+
+      @include responsive(T) {
+        @include column-flexbox($gap: 0);
```

```suggestion
    .section-bottom {
      @include flexbox(between, start);

      @include responsive(T) {
        @include column-flexbox;
```
다른 사이즈일 때 gap이 적용되지 않았으니까 0으로 없애주지 않아도 될 거 같습니다